run/URBPARM_LCZ.TBL

```diff
@@ -468,41 +468,7 @@ HSEQUIP: 0.25 0.25 0.25 0.25 0.25 0.25 0.25 0.5 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0
 HSEQUIP_SCALE_FACTOR: 36.00, 20.00, 20.00, 36.00, 20.00, 20.00, 20.00, 36.00, 20.00, 20.00, 20.00 
 
 
-#
-# GR_FLAG: 1 to switch on green roof model (0-1)
-#      (sf_urban_physics=3)
-#
-
-GR_FLAG:0
-
-#
-# GR_TYPE: 1 for GRASS, 2 for SEDUM vegetation on the green roof
-#      (sf_urban_physics=3)
-#
-
-GR_TYPE: 2
-
-#
-# GR_FRAC_ROOF: fraction of green roof over the roof (0:1)
-#      (sf_urban_physics=3)
-#
-
-GR_FRAC_ROOF:0,0,0,0,0,0,0,0,0,0,0
-
-#
-# HSEQUIP:  Diurnal  profile of sprinkler irrigation for the green roof 
-#      (sf_urban_physics=3)
-#
-
-IRHO:0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1
-
-
-#
-# PV_FRAC_ROOF: fraction of photovoltaic panels over the roof (0:1)
-#      (sf_urban_physics=3)
-#
 
-PV_FRAC_ROOF: 0,0,0,0,0,0,0,0,0,0,0
```

Comment:
Is the intent to remove all of this green roof, photovoltaic info?

Reply:
definitely not. This info must have been missing in the file I originally received from Andrea Zonato.

Reply:
The green roof and photovoltaic info should not be removed.

Reply:
> @weiwangncar @dudhia @matthiasdemuzere
> Wei and Jimy,
> The PR seems OK to review. Who else should be asked to look over these changes? Do we request a difference plot?

You could ask Andrea Zonato. I already did, and he approved, but hard to say whether he double-ckecked all values in detailM

> Matthias,
> Looking though the changes, most are small modifications to the value, such as 10 -> 12. But there are a couple in the first two lines of changes that have now been set to zero (from 350, and from 25). These are OK?

Indeed, some values have large changes. These occur at position 11, which refers to LCZ class 15/E, bare rock or paved, see [here](https://www.wudapt.org/lcz/lczs-a-to-f/). Setting this class as urban is ambiguous, as this also refers to natural bare rock. See example [this LCZ map for Las Vegas](https://lcz-generator.rub.de/factsheets/259842311c49a312ae1a85e97738897d8564e038/259842311c49a312ae1a85e97738897d8564e038_factsheet.html), were much of the natural surroundings is LCZ E (black areas), which would be assigned with "urban" properties.

In a way, this will be addressed with a new pre-processor we are developing. Yet nevertheless, I want to make sure some values are really hard-coded to 0, such as AH/ALH (there is no anthropogenic heat from these surfaces).

Hope this clarifies these changes?